cli/run.go

```diff
@@ -403,8 +403,13 @@ func createWorkflowRunPlan(modes models.WorkflowRunModes, targetWorkflow string,
 		})
 	}
 
+	cliVersion := version.VERSION
+	if version.IsAlternativeInstallation {
```

Comment:
Would it make sense to test if the commit isn't empty?

Reply:
I did not write tests because this part of the cli is not covered with unit tests and writing one would have taken a lot of time. Same for the integration tests as this only appears in the build output and I had to build a special binary version just for this where I control the `Commit` value seemed like a lot of investment. And this feature is really not critical in the sense that the absence of it would not impact anything.

Based on these I thought it is fine for now to not to write test for this.